default.html.j2

```diff
@@ -240,64 +260,61 @@
 {# ------------------------------------------------------------------------------------ #}
 {#  CHAPTER MACROS                                                                      #}
 {# ------------------------------------------------------------------------------------ #}
-
-{%- macro renderChapterReport(chapter) -%}
+{%- macro renderReport(report) -%}
   <section class="report">
-    <h3 class="title">Report</h3>
-
-    <h4>Indications</h4>
+    <h3 class="title">Summary</h3>
     <div class="indications">
       <table>
         <tbody>
-          {% for chapterReport in report.chapterReports %}
-            {% if chapterReport.chapterUuid == chapter.uuid %}
-              {% for indication in chapterReport.indications %}
-                <tr>
-                  <td>Answered {% if indication.indicationType == "PhasesAnsweredIndication" %}(current phase){% endif %}</td>
-                  <td>{{indication.answeredQuestions}} / {{indication.answeredQuestions + indication.unansweredQuestions}} </td>
-                </tr>
-              {% endfor %}
-            {% endif %}
+          {% for indication in report.indications %}
+            <tr>
+              <td class="table-label">Answered {% if indication.indicationType == "PhasesAnsweredIndication" %}(current phase){% endif %}</td>
+              <td>{{indication.answeredQuestions}} / {{indication.answeredQuestions + indication.unansweredQuestions}} </td>
+              <td class="progress-cell">
+                <div class="progress">
+                  <div class="progress-fill" style="width:{{100 * indication.answeredQuestions / (indication.answeredQuestions + indication.unansweredQuestions)}}%">
```

Comment:
This may end up with `ZeroDivisionError: division by zero`, especially if there are no desirable question for a phase... best would be to do something like this:

```html
{% set total = indication.answeredQuestions + indication.unansweredQuestions %}
...
<div class="progress-fill" style="width:{{100 * indication.answeredQuestions / [1, total]|max}}%">
...
```

Or handle it using condition...